src/services/administration/iam/role/update-role/modules/RoleUpdatePageBaseInformation.vue

```diff
@@ -95,6 +95,12 @@ export default {
         watch(() => isAllValid.value, (after) => {
             emit('update-validation', after);
         }, { immediate: true });
+        watch([() => state.selectedRoleType, () => state.roleDescription, () => roleName.value], (after) => {
```

Comment:
```suggestion
        watch([() => state.selectedRoleType, () => state.roleDescription, () => roleName.value], ([selectedRoleType, roleDescription, roleName]) => {
```

이렇게 받아오면 좀더 명확할 거 같아요!

Reply:
전개 전개
감사합니다